templates/vsftpd.conf.erb

```diff
@@ -24,82 +24,51 @@ local_umask=<%= @local_umask %>
 # Uncomment this to allow the anonymous FTP user to upload files. This only
 # has an effect if the above global write enable is activated. Also, you will
 # obviously need to create a directory writable by the FTP user.
-<% if @anon_upload_enable == 'YES' -%>
-anon_upload_enable=YES
-<% else -%>
-#anon_upload_enable=YES
-<% end -%>
+#anon_upload_enable=<%= @anon_upload_enable %>
 #
 # Uncomment this if you want the anonymous FTP user to be able to create
 # new directories.
-<% if @anon_mkdir_write_enable == 'YES' -%>
-anon_mkdir_write_enable=YES
-<% else -%>
-#anon_mkdir_write_enable=YES
-<% end -%>
+#anon_mkdir_write_enable=<%= @anon_mkdir_write_enable %>
 #
 # Activate directory messages - messages given to remote users when they
 # go into a certain directory.
 dirmessage_enable=<%= @dirmessage_enable %>
 #
-# Activate logging of uploads/downloads.
+# The target log file can be vsftpd_log_file or xferlog_file.
+# This depends on setting xferlog_std_format parameter
 xferlog_enable=<%= @xferlog_enable %>
 #
 # Make sure PORT transfer connections originate from port 20 (ftp-data).
-connect_from_port_20=<%= @connect_from_port_20 %>
+#connect_from_port_20=<%= @connect_from_port_20 %> 
```

Comment:
Again mentioned changes are as per what we have on current production template for sftp/ftp environment.